test/unit/services/instanceAdapter_spec.js

```diff
@@ -22,11 +22,29 @@ describe("The InstanceAdapter class ", function () {
 
 			instances.createInstance()
 			.then(function (result) {
+				var failedInstance = new Instance({
+					id    : result.id,
+					ami   : result.ami,
+					state : "failed",
+					type  : result.type,
+					uri   : result.uri
+				});
+
 				expect(result).to.be.an.instanceOf(Instance);
 				expect(result.id).to.match(ID_REGEX);
 				expect(result.type).to.equal(DEFAULT_TYPE);
 				expect(result.ami).to.equal(DEFAULT_AMI);
+
+				return instances.updateInstance(failedInstance)
```

Comment:
What is the rationale for calling `updateInstance()` in the test case for creating instances?